apps/client/src/widgets/community/components/detail-section/detail-section.tsx

```diff
@@ -26,10 +26,14 @@ const DetailSection = ({ postId }: DetailSectionProps) => {
 
   const [imageFile, setImageFile] = useState<File | null>(null);
   const [imagePreview, setImagePreview] = useState<string | null>(null);
+  const prevModeRef = useRef(mode);
 
   const queryClient = useQueryClient();
 
   useEffect(() => {
+    const prevMode = prevModeRef.current;
+    prevModeRef.current = mode;
```

Comment:
```suggestion
    prevModeRef.current.current = mode;
```
이거랑 똑같다면 변수 한 단계 줄이는것에 대해서 어떻게 생각해용?

Reply:
줄여도 상관 없을것 같네요! 바로 반영하겠스빈다

Reply:
추가적으로 current.current 로 접근하는 것에 대해서도 deps 를 줄일 수 있다면 best